pyproject.toml

```diff
@@ -52,7 +52,7 @@ ci = [
 check = [
     "build",
     "pre-commit",
-    "docstub @ git+https://github.com/scientific-python/docstub@8c35c82585cd50472161abf3a8e191da1c509abf",
+    "docstub>=0.4",
```

Comment:
given the main goal of this is running the pre-commit check I think we should pin it

```suggestion
    "docstub==0.4",
```